src/java/org/apache/cassandra/locator/InetAddressAndPort.java

```diff
@@ -122,6 +122,12 @@ public String getHostAddress(boolean withPort)
         return hostAddress(this, withPort);
     }
 
+    public String getHostName(boolean withPort)
+    {
+        StringBuilder sb = new StringBuilder(getHostName());
```

Comment:
This can be just simplified to this:

`return withPort ? String.format("%s:%s", getHostname(), getPort()) : getHostname();`

I do not think we need `StringBuilder` for this.

Reply:
@smiklosovic,
I've committed the new return statement at [adb93aa](https://github.com/apache/cassandra/pull/1910/commits/adb93aaa5722e6bd12b0b5e6eaaa864d511049c9) and seen that `InetAddressAndPortTest.java` unit test passed.